tfhe/src/core_crypto/commons/ciphertext_modulus.rs

```diff
@@ -103,32 +167,41 @@ impl<Scalar: UnsignedInteger> CiphertextModulus<Scalar> {
     /// implementations than can rely on wrapping arithmetic operations behavior to compute the
     /// modulus.
     pub const fn is_native_modulus(&self) -> bool {
-        if self.0 == 0 {
-            return true;
+        match self.inner {
```

Comment:
could use `matches!(self.inner, Self::Native)`
https://doc.rust-lang.org/std/macro.matches.html

Reply:
ah right, I never think about that matches macro